ApiaryReportingApi.apib

```diff
@@ -64,406 +66,365 @@ HOST: http://api.apiary.io/
-          }],
-          "public": "true",
-          "agent": "Dredd/0.1.7 (Linux 3.2.0-23-generic; x64)",
-          "agentRunUuid": "03b3de07-d354-4ba6-aa2e-28789b03c619",
-          "hostname": "machine.fqdn.tld",
-          "agentEnvironment": {
-            "CI": "true"
-          },
-          "startedAt": 1381924285,
-          "endedAt": "1381924299",
-          "status": "passed",
-          "result": {
-            "tests": 7,
-            "failures": 0,
-            "passes": 7
-          }
-        }
-        ```
 
 
 ## Group Test Steps
 
 - Attributes
-  - _id (string) - Unique object id. DO NOT USE IT WHEN CREATING! Used in post body only for purpose of testing this blueprint.
-  - testRunId (string) - Reference to the parent test run identificator
-  - origin (object) - Origin path (position, location) of the step in the blueprint AST
-    - filename (string) .. File name of original blueprint
-    - uriTemplate (string) - URI Template used as source for URL of this step
-    - resourceGroupName (string) - Group name
-    - resourceName (string) - Resource name
-    - actionName (string) - Action name
-    - exampleName (string) - Example name
-  - duration (number) - Test duration in seconds
-  - result (string) - Result of step execution
-    - Values
-      - `passed`
-      - `failed`
-  - resultData (object) - Data from step execution
-    - request  - [Real HTTP Request](https://www.relishapp.com/apiary/gavel/v/0-1/docs/data-model#http-request)
-    - realResponse - [Real HTTP response](https://www.relishapp.com/apiary/gavel/v/0-1/docs/data-model#http-response)
-    - expected Response - [Expected HTTP Response](https://www.relishapp.com/apiary/gavel/v/0-1/docs/data-model#expected-http-response)
-    - result - [Validation result output](https://www.relishapp.com/apiary/gavel/v/0-1/docs/data-validators-and-output-format#validators-output-format)
-  - stepType (string, `exampleNameTransactionName`) - Type of the step, should be HTTP or Cucumber for instance
+    - _id (string) - Unique object id. DO NOT USE IT WHEN CREATING! Used in post body only for purpose of testing this blueprint.
+    - testRunId (string) - Reference to the parent test run identificator
+    - origin (object) - Origin path (position, location) of the step in the blueprint AST
+        - filename (string) .. File name of original blueprint
+        - uriTemplate (string) - URI Template used as source for URL of this step
+        - resourceGroupName (string) - Group name
+        - resourceName (string) - Resource name
+        - actionName (string) - Action name
+        - exampleName (string) - Example name
+    - duration (number) - Test duration in seconds
+    - result (string) - Result of step execution
+        - Values
+            - `passed`
+            - `failed`
+    - resultData (object) - Data from step execution
+        - request  - [Real HTTP Request](https://www.relishapp.com/apiary/gavel/v/0-1/docs/data-model#http-request)
```

Comment:
I am bit confused by square brackets here (`[]`); it's probably my missing knowledge, but I'd like to clarify before merging.